server/db/015-addnotifications.sql

```diff
@@ -0,0 +1,42 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 15;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        CREATE TABLE IF NOT EXISTS notifications (
+            id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+            installation_id uuid REFERENCES installations (id),
```

Comment:
I think that `installation_id` needs an explicit `NOT NULL` (https://stackoverflow.com/questions/28206232/can-foreign-key-references-contain-null-values-in-postgresql)

Reply:
gotcha, good catch.